src/index.css

```
@@ -157,122 +157,224 @@ body {
-  min-height: clamp(220px, 40vh, 360px);
+/* Visually hidden, but read by assistive tech. Used for spoken separators and labels
+   that carry meaning a sighted reader gets from layout or a tooltip. */
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)